src/main/java/com/google/javascript/gents/TypeScriptGenerator.java

```diff
@@ -176,26 +183,42 @@ private void generateTypeScript() {
 
     new StyleFixPass(compiler, comments).process(externRoot, srcRoot);
 
+    new AstValidator(
+            compiler,
+            new AstValidator.ViolationHandler() {
+              @Override
+              public void handleViolation(String message, Node n) {
+                compiler.report(JSError.make(n, CLUTZ_MALFORMED_AST, message));
```

Comment:
This isn't that helpful, actually. It just spits out errors with no context:

```
ERROR - Clutz produced a malformed AST: Expected 0 children, but was 1

ERROR - Clutz produced a malformed AST: Expected EMPTY but was NAME

ERROR - Clutz produced a malformed AST: Expected expression but was EXPORT
```

I'm not sure if that's more useful than nothing, unfortunately. Is there any way to get more detail here? At least a filename, and ideally a location?